### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by guizmaii**
The fun fact is that I noticed this issue because it's something we fixed on our app code several months ago :)
But I'm not sure this applies to zio-kafka code, not knowing it very well yet (even if I really start to understand how the `RunLoop` works).
That's why, I still have to write the test but, I'd like your review first @svroonland

### CODE REVIEW

File: zio-kafka/src/main/scala/zio/kafka/consumer/internal/Runloop.scala [hunk 1]
```diff
@@ -548,7 +548,7 @@ private[consumer] final class Runloop(
   def run: ZIO[Scope, Nothing, Fiber.Runtime[Throwable, Unit]] =
     ZStream
       .mergeAll(3, 1)(
-        ZStream(Command.Poll).repeat(Schedule.spaced(pollFrequency)),
+        ZStream(Command.Poll).repeat(Schedule.once ++ Schedule.spaced(pollFrequency)),
```
Comment (svroonland) [inline zio-kafka/src/main/scala/zio/kafka/consumer/internal/Runloop.scala:551]: I had to look twice, but I believe `ZStream(x).repeat(schedule)` immediately emits `x` and afterwards according to the `schedule`, which is what we want isn't it?

Comment (guizmaii) [reply zio-kafka/src/main/scala/zio/kafka/consumer/internal/Runloop.scala:551]: I'll write a test to be validate the behaviour of `spaced`

Comment (guizmaii) [reply zio-kafka/src/main/scala/zio/kafka/consumer/internal/Runloop.scala]: FYI @svroonland, I was wrong. The code was indeed already emitting a first `Poll` immediately (my tests are proving this)

But, as mentioned here https://github.com/zio/zio-kafka/issues/428#issuecomment-1445352019, I think we should change this `ZStream(Command.Poll).repeat(Schedule.spaced(pollFrequency))` piece of code by `ZStream.repeatWithSchedule(Command.Poll, Schedule.fixed(pollFrequency))` for 2 reasons:

1. I think that `ZStream.repeatWithSchedule(Command.Poll, ...)` is easier to reason about compared to ` ZStream(Command.Poll).repeat(...)`

In the `ZStream(Command.Poll).repeat(...)` version, it's IMO hard to understand if `ZStream(Command.Poll)` (without the `.repeat(...)` part is emitting this first `Poll` event. And if it does so, if the `.repeat(...)` part triggers an immediate event too? Hard to tell and understand IMO
While the `ZStream.repeatWithSchedule(Command.Poll, ...)` is IMO more explicit. It's trigger this `Command.Poll` according to the passed `Schedule`. No ambiguity.

2. I think that with `Schedule.spaced` we'll always waiting `pollInterval` even if this `pollInterval` has been spend doing something else. With `Schedule.fixed`, if this `pollInterval` amount of time has been spent doing something else, the `Poll` will be triggered immediatly

Comment (svroonland) [reply zio-kafka/src/main/scala/zio/kafka/consumer/internal/Runloop.scala]: Agreed, `ZStream.repeatWithSchedule(Command.Poll, Schedule.fixed(pollInterval)` is better for now, until we implement the suggestions in #664.

**Pull request closed without merging**